lib/tasks/exercise_test_tasks.rb

```diff
@@ -16,18 +16,16 @@ def initialize(options:, test_runner: ExerciseTestsRunner)
   def define
     define_task_for_all_exercises
 
-    exercises.each { |exercise| define_task_for(exercise) }
+    #Exercise.concept.each { |exercise| define_task_for(exercise) }
+    Exercise.practice.each { |exercise| define_task_for(exercise) }
   end
 
   private
 
-  def exercises
-    @exercises ||= Exercise.all
-  end
-
   def define_task_for_all_exercises
     desc 'Run the tests for all exercises'
-    task exercises: exercises
+    # TODO: Also add concept exercises
+    task exercises: Exercise.practice
```

Comment:
I've tested this with `Exercise.all` and it works in terms of generating the tests correctly, but the concept exercises currently fail the tests due to the structure being different, so I've stuck to practice exercises for now.